docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -103,7 +190,149 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use YAML file ``values.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
```

Comment:
I really thought about this but Jinja templating is involved enough that it should have its own documentation. I could probably modify it to present another error type or 2 if that would help

Reply:
I think `realizing` a config is exactly where it should be described. That's where we fill in those jinja templates and then overwrite values provided by supplemental files. Both of those actions should be documented here in a way that shows what is actually happening. Otherwise `realizing` is only copying one file format to another, which is something we're discussing taking away.

Reply:
Ah thanks I misunderstood, I was thinking about `translate` for some reason - working on it!

Reply:
Hm, this version is still kind of simplistic but a little better - let me know if you think its still too short

Reply:
I'd like to see that some of those Jinja2 templates are filled in for a completely realized output. It probably makes things too SRW-specific to use this example, but check out Slide 17 in [this slide deck](https://docs.google.com/presentation/d/1GziWguL2ivZo-VlgPcZi13bxcNvJ1fwJZKMwGixYrQQ/edit#slide=id.g1a118e06cb4_0_18).

Reply:
Sure, I can fill in some of the examples in a less-SRW-y way. You're right that it really should show full rendering

Reply:
What do you think of this:
`values.yaml`
```
values:
  greeting: Hello
  recipient: World
  message: '{{ (GREETING + RECIPIENT) * REPEAT }}'
  repeat: 1
```
`config.yaml`
```
values:
  greeting = Good Night
  recipient = Moon
  repeat: 2
```
However, its not rendering when I run `realize`

Reply:
Ah, case-sensitivity fixes it - will add shortly!